includes/REST_API/Stories_Controller.php

```diff
@@ -99,6 +102,11 @@ public function prepare_item_for_response( $post, $request ) {
 			$data['poster_portrait_url'] = ! empty( $image ) ? $image : $schema['properties']['featured_media_url']['default'];
 		}
 
+		if ( in_array( 'color_presets', $fields, true ) ) {
+			$color_presets         = get_option( self::COLOR_PRESETS_OPTION, [] );
+			$data['color_presets'] = is_array( $color_presets ) ? $color_presets : [];
```

Comment:
Why not just cast to an array here?

Reply:
Since if it's not an array then likely something is wrong there and it might fail in the FE anyway. Safer to send an empty array if sth went wrong.